src/main/java/com/google/finapp/SpannerDaoJdbcImpl.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Google LLC
```

Comment:
I'd leave this file out for now, we can wait for Nick's PR.

Instead of creating the jdbc object you can just throw an exception that it's not implemented and we can add it later what do you think?

Reply:
so something like...
`if (SpannerDaoImpl == "jdbc") { throw [some exception]; }`?